docs/src/guides/languages/rust.md

```diff
@@ -9,7 +9,7 @@ tags:
 # :simple-rust: Rust
 <!-- markdownlint-enable single-h1 -->
 
-<!-- cspell: words toolsets stdcfgs depgraph -->
+<!-- cspell: words toolsets stdcfgs depgraph Rustup -->
```

Comment:
Can we move all these words to [project.dic](https://github.com/input-output-hk/catalyst-ci/pull/232/files#diff-c74e62111a947fe7699747e1193dd592b3346ab2267f7db4c78c994b2569cd49)?